operator/iamkeycontroller/create.go

```diff
@@ -74,7 +74,6 @@ func (p *IAMKeyPipeline) createIAMKey(ctx *pipelineContext) error {
 		// Some permissions are excluded such as create and list all sos buckets.
 		exoscalesdk.CreateIAMAccessKeyWithOperations(IAMKeyAllowedOperations),
 		// The tag and resource are used to limit the permissions to object storage
```

Comment:
Remove this line as well, it is not needed anymore :)

Reply:
A fair, thanks. Half of the comment is wrong now :sweat_smile: